dist/model.js

```diff
@@ -0,0 +1,42 @@
+class Model{ 
+    constructor(){
+        this.cityData = []
+
+    }
+
+    async getDataFromDB(){
```

Comment:
I would call this instead `getCities` that describes better when the method does.